package.json

```diff
@@ -80,7 +80,7 @@
     "regenerator-runtime": "^0.11.1",
     "semantic-release": "^15.5.0",
     "static-extend": "^0.1.2",
-    "styled-components": "^3.4.5",
+    "styled-components": "^4.1.2",
```

Comment:
My memory is a bit fuzzy, but for some reason I feel like we need to take a hard look at updating force to the latest styled-components before we can implement elsewhere.